.env

```diff
@@ -1,14 +1,8 @@
-# Environment variables declared in this file are automatically made available to Prisma.
-# See the documentation for more detail: https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
-
-# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
-# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
-
-
-#DATABASE_URL='mysql://9tgkpvfgkmhft1linvsv:************@aws.connect.psdb.cloud/vision-aid-prototype?sslaccept=strict'
-
 DATABASE_URL=mysql://root:root@localhost:3306/vision
 
 NEXTAUTH_URL=http://localhost:3000
 NEXT_PUBLIC_NEXTAUTH_URL=http://localhost:3000
 NEXTAUTH_SECRET=aBNRWEZmLB9qbdFy3fxcikKQP5VIRsEzJcVfUwUfi1g=
+
+GOOGLE_CLIENT_SECRET=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
+GOOGLE_CLIENT_ID=177788693840-psdvlsd87spkefasfqrtr00l0928q2ke.apps.googleusercontent.com
```

Comment:
Please remove this file. We are no longer checking it into Github. You do need a local copy however.

I have the changes in a local branch, but if you'd like you can update the `.gitignore` to include this file. This way, Github wont recognize changes that you make and this won't happen again